girder_style/templates/account/base.html

```diff
@@ -5,17 +5,18 @@
 {% block body %}
 <body class="bg-gray-100 flex flex-col h-full relative">
 
-<div class="absolute bg-black bottom-0 left-0 overflow-hidden right-0 top-0 z-0">
+<div class="absolute bg-black bottom-0 left-1/4 overflow-hidden right-0 bottom-0 top-0 z-0">
   <img
-      src="https://images.pexels.com/photos/53176/architecture-iron-steel-building-53176.jpeg?cs=srgb&dl=pexels-pixabay-53176.jpg&fm=jpg"
-      class="object-cover opacity-25 w-full"
+      src="https://images.unsplash.com/photo-1543996991-8e851c2dc841?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=2251&q=80"
+      class="hidden object-cover opacity-50 h-full w-full md:block"
       alt="">
 </div>
 
 <div class="flex flex-initial h-24 items-center justify-between relative w-full z-20">
   <div class="bg-white flex h-full items-center max-w-max px-6 py-4 w-full lg:w-1/2 md:max-w-xl md:px-12 md:w-2/3">
-    <div class="max-w-3xs">
-      <img src="https://blog.kitware.com/wp-content/uploads/2016/03/Girder_v2.png" class="w-full h-auto" alt="Girder">
+    <div class="flex items-center h-full text-xl font-bold">
+      <img src="https://resonant.kitware.com/wp-content/uploads/2018/10/Girder_Mark.svg" class="h-full mr-2 w-auto" alt="Logo Mark Name">
```

Comment:
```suggestion
      <img src="https://resonant.kitware.com/wp-content/uploads/2018/10/Girder_Mark.svg" class="h-full mr-2 w-auto" alt="Logo Mark">
```

This logo would typically not include the name, right?